lib/school.rb

```diff
@@ -0,0 +1,32 @@
+class School
+  attr_reader :start_time,
+              :hours,
+              :student_name
+
+  def initialize(start_time, hours)
+    @start_time = start_time
+    @hours = hours
+
+end
+
+school = School.new('9:00', 7)
```

Comment:
You shouldn't have any executable code in your class file - and trying to create an instance of a class from within the class itself is generally going to cause some problems.